test/uk/gov/hmrc/customs/emailfrontend/connectors/CustomsDataStoreConnectorSpec.scala

```diff
@@ -16,51 +16,56 @@
 
 package uk.gov.hmrc.customs.emailfrontend.connectors
 
-import org.mockito.ArgumentMatchers.{eq => meq}
 import org.scalatest.BeforeAndAfterEach
 import play.api.http.Status.{NOT_FOUND, NO_CONTENT}
 import play.api.libs.json.Json
 import uk.gov.hmrc.customs.emailfrontend.audit.Auditable
 import uk.gov.hmrc.customs.emailfrontend.config.AppConfig
-import uk.gov.hmrc.customs.emailfrontend.connectors.http.responses._
+import uk.gov.hmrc.customs.emailfrontend.connectors.http.responses.*
 import uk.gov.hmrc.customs.emailfrontend.model.{Eori, UpdateEmail}
 import uk.gov.hmrc.customs.emailfrontend.services.DateTimeService
 import uk.gov.hmrc.customs.emailfrontend.utils.TestData.dateFormatter02
 import uk.gov.hmrc.customs.emailfrontend.utils.SpecBase
 import uk.gov.hmrc.customs.emailfrontend.utils.Utils.emptyString
-import uk.gov.hmrc.http.{BadRequestException, HeaderCarrier, HttpClient, HttpResponse, InternalServerException}
+import uk.gov.hmrc.http.client.{HttpClientV2, RequestBuilder}
+import uk.gov.hmrc.http.{BadRequestException, HeaderCarrier, HttpReads, HttpResponse, InternalServerException}
+import org.mockito.Mockito.{doNothing, reset, when}
+import org.mockito.ArgumentMatchers.any
 
 import java.time.LocalDateTime
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.{ExecutionContext, Future}
 
 class CustomsDataStoreConnectorSpec extends SpecBase with BeforeAndAfterEach {
 
-  private val mockHttp = mock[HttpClient]
+  private val mockHttp = mock[HttpClientV2]
+  private val requestBuilder = mock[RequestBuilder]
   private val mockAuditable = mock[Auditable]
   private val mockAppConfig = mock[AppConfig]
   private val mockDateTimeService = mock[DateTimeService]
+
   private implicit val hc: HeaderCarrier = HeaderCarrier()
   private val connector = new CustomsDataStoreConnector(mockAppConfig, mockHttp, mockAuditable)
-  private val url = "/customs-data-store/update-email"
+  private val postUrl = "http://localhost:9893/customs-data-store/update-email"
   private val testEori: Eori = Eori("GB1234556789")
   private val testEmail = "email@test.com"
   private val testDateTime = LocalDateTime.parse("2021-01-01T11:11:11.111Z", dateFormatter02)
-  private val requestBody: UpdateEmail = UpdateEmail(testEori, testEmail, testDateTime)
-  private val headers = Seq("Content-Type" -> "application/json")
 
   override def beforeEach(): Unit = {
-    reset(mockHttp, mockAuditable, mockAppConfig)
-    when(mockAppConfig.customsDataStoreUrl).thenReturn(url)
+    reset(mockHttp, mockAuditable, mockAppConfig, requestBuilder)
```

Comment:
Please add a line break after this line to make it more readable and to separate from the when statements